common/djangoapps/student/views/management.py

```diff
@@ -422,6 +423,8 @@ def change_enrollment(request, check_access=True):
                 enroll_mode = CourseMode.auto_enroll_mode(course_id, available_modes)
                 if enroll_mode:
                     CourseEnrollment.enroll(user, course_id, check_access=check_access, mode=enroll_mode)
+            except EnrollmentNotAllowed as exec:  # pylint: disable=broad-except
```

Comment:
I'd suggest using `as e` instead of `exec` so pylint doesn't think we're referring to exec. Also, why is the pylint disable needed?